weather_mv/loader_pipeline/ee.py

```diff
@@ -294,8 +303,26 @@ def validate_arguments(cls, known_args: argparse.Namespace, pipeline_args: t.Lis
                             region=pipeline_options_dict.get('region'))
             logger.info('Region validation completed successfully.')
 
+        # Check for the band_names json file
+        if not os.path.exists(known_args.band_names):
+            raise RuntimeError("--band_names should contain a valid file that exists")
+        else:
+            if not known_args.band_names.split('.')[-1] == 'json':
+                raise RuntimeError("--band_names should contain a json file as input")
+
+        # Check the initialization_time and forecast_time strings
+        chars = ['%Y', '%m', '%d', '%H', '%M', '%S']
+        if not all([char in known_args.initialization_time for char in chars]):
```

Comment:
Should we check if all these are of type `int` in the `uri`?

Because we are just checking if these chars are present in regex but if someone provides a wrong regex (with all these chars) then `re.findall` will not find anything and `match_datetime()` function will crash. I'd prefer to check those here rather than managing it in the `match_datetime()` function.